src/Avalonia.Base/Point.cs

```diff
@@ -164,6 +164,19 @@ public Point(double x, double y)
         /// <returns>The resulting point.</returns>
         public static Point operator *(Point point, Matrix matrix) => matrix.Transform(point);
 
+        /// <summary>
+        /// Computes the Euclidean distance between the two given points.
+        /// </summary>
+        /// <param name="value1">The first point.</param>
+        /// <param name="value2">The second point.</param>
+        /// <returns>The Euclidean distance.</returns>
+        public static double Distance(Point value1, Point value2)
+        {
+            double distanceSquared = ((value2.X - value1.X) * (value2.X - value1.X)) +
```

Comment:
I suggest using `Math.Pow` here.

Reply:
Why? As noted in the PR description Math.Pow() is an unnecessary overhead here. The code is like this for performance.